reformat/fix_ft_int_issue.py

```diff
@@ -0,0 +1,174 @@
+##############
+# Iterate through one file
+with open(ft_input, 'r') as fh, open(ft_output, 'w') as out:
+    irow = 0
+    for line in fh:
+        # Write col headers to out
+        if irow == 0:
+            out.write(line)
+            irow+=1
+        else:
+            line = line.strip().split('\t')
+            model = line[0].strip("\"")
+            tumor = line[1]
+            ftset = line[2]
+
+            # Reformat so clear string
+            model = 'model' + model
+            out.write(model + '\t')
+
+            # No reformat to tumor
+            out.write(tumor + '\t')
+
+            # no reformat ftset
+            out.write(ftset + "\n")
+
+fh.close()
+out.close()
+
+
+
+############## Prediction files
+ft_input = '/Users/leejor/Ellrott_Lab/gdan-tmp-webdash/data/GNOSIS_FIRST-predictions/'
+ft_output = '/Users/leejor/Ellrott_Lab/gdan-tmp-webdash/data/predictions/'
+##############
+import os
+import glob
+
+# Read in all feature files from input dir
+os.chdir(ft_input)
+files = glob.glob('reformat*single_sample_predictions.tsv')
+
+
+os.chdir(ft_output)
+# Iterate through all files in a dir
+for f in files:
+    # set up file names
+    a = f.strip()
+    outputname = "fixattempt2--" + a
+
+    # Reformat and save tsv outputs
+    with open(ft_input+f, 'r') as fh, open(outputname, 'w') as out:
+        irow = 0
+        for line in fh:
+            # fix col headers
+            if irow == 0:
+                line = line.strip().split('\t')
+                nitems = len(line)
+                s = line[0]
+                r = line[1]
+                f = line[2]
+                t = line[3]
+                l = line[4]
+
+                # No reformat
+                out.write(s + '\t')
+                out.write(r + '\t')
+                out.write(f + '\t')
+                out.write(t + '\t')
+                out.write(l)
+
+                # reformat all models
+                for i in range(5, nitems):
+                    header = line[i]
+                    classifer = header.split('|')[0]
+                    ftmethod = header.split('|')[1].strip("\"")
+                    temporal = header.split('|')[2]
+                    crisp = header.split('|')[3]
+
+                    #add "model" infront of model int
+                    ftmethod = "model" + ftmethod
+
+                    #fix temporal. 2020-03-13 08:17:44.059 -> 2020-03-13T08:17:44.059
+                    date = temporal.split(" ")[0]
+                    clock = temporal.split(" ")[1]
+                    temporal = date + "T" + clock
+
+                    newline = classifer+ "|"+ftmethod+"|"+temporal+"|"+crisp
+                    out.write('\t' + newline)
+                out.write('\n')
+                irow+=1
+            #Write all non col headers to out
+            else:
+                out.write(line)
+
+
+    fh.close()
+    out.close()
```

Comment:
The `.close` is needed because you opened the files using the `with` command

Reply:
rm unnecessary script but incorp these changes into scripts called on by wrappers in /reformat/